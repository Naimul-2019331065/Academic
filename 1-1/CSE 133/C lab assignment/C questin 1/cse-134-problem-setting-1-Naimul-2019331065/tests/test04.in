10
20 25
2 3
4 0
2 5
0 4
2 5
7 9
0 0
16 10
0 0
5 7
5 0
7 5
5 0
5 7
0 5
7 5
0 5
188 6
5 7
