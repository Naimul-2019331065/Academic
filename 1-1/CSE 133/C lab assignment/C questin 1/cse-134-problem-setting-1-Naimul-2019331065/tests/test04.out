10
0
0
9
10
7
5
1
1
6